.hlint-universum.yaml

```diff
@@ -0,0 +1,1050 @@
+############################################################################
+## Universum
+############################################################################
+
+# There's no 'head' in Universum
+- ignore: {name: "Use head"}
+
+# We have 'whenJust' for this
+- ignore: {name: "Use Foldable.forM_"}
+
+- warn: {lhs: Data.Text.pack, rhs: Universum.toText}
+- warn: {lhs: Data.Text.unpack, rhs: Universum.toString}
+
+- warn: {lhs: Data.Text.Lazy.pack, rhs: Universum.toLText}
+- warn: {lhs: Data.Text.Lazy.unpack, rhs: Universum.toString}
+- warn: {lhs: Data.Text.Lazy.toStrict, rhs: Universum.toText}
+- warn: {lhs: Data.Text.Lazy.fromStrict, rhs: Universum.toLText}
+
+- warn: {lhs: Data.Text.pack (show x), rhs: Universum.show x}
+- warn: {lhs: Data.Text.Lazy.pack (show x), rhs: Universum.show x}
+
+- warn: {lhs: Control.Exception.evaluate, rhs: evaluateWHNF}
+- warn: {lhs: Control.Exception.evaluate (force x), rhs: evaluateNF x}
+- warn: {lhs: Control.Exception.evaluate (x `deepseq` ()), rhs: evaluateNF_ x}
+- warn: {lhs: void (evaluateWHNF x), rhs: evaluateWHNF_ x}
+- warn: {lhs: void (evaluateNF x), rhs: evaluateNF_ x}
+
+## Monad.Trans
```

Comment:
Okay, let's discuss. I think this stuff is too non-standard and we shouldn't force people to use it.

Reply:
I probably agree with you on this one. _However_, this will have to be reconciled with the fact that these functions still remain in Universum. Perhaps we should deprecate them?

Reply:
I generally don't think that we should force to much. If some function exists, we shouldn't necessarily require using it whenever possible or deprecate it instead. Too severe limitations are usually bad.
If some person likes `usingReaderT`, I'd let them use it, but if they don't, `flip runReaderT` should be acceptable as well. I don't think that one of these options is so much better than another one.